assets/base.css

```diff
@@ -483,11 +483,6 @@ blockquote {
   letter-spacing: 0.04rem;
 }
 
-h2.caption-large,
-h3.caption-large {
-  font-size: calc(var(--font-heading-scale) * 1.3rem);
-}
-
 .color-foreground {
   color: rgb(var(--color-foreground));
 }
```

Comment:
`text-body` doesn't seem to be adding anything here: https://screenshot.click/19-28-raug7-xx9ui.mp4

Reply:
By default, the theme's body and heading font is the same. If we set a different heading font, the `text-body` will ensure the element font-family is that of body text.